extension/src/setup/webview/messages.ts

```diff
@@ -21,19 +27,53 @@ export class WebviewMessages {
   public sendWebviewMessage(
     cliAccessible: boolean,
     projectInitialized: boolean,
-    hasData: boolean
+    isPythonExtensionInstalled: boolean,
+    pythonBinPath: string | undefined,
+    hasData: boolean | undefined
   ) {
     this.getWebview()?.show({
       cliAccessible,
       hasData,
-      projectInitialized
+      isPythonExtensionInstalled,
+      projectInitialized,
+      pythonBinPath
     })
   }
 
   public handleMessageFromWebview(message: MessageFromWebview) {
     if (message.type === MessageFromWebviewType.INITIALIZE_PROJECT) {
       return this.initializeProject()
```

Comment:
[I] This still needs analytics attached. Probably should be done in the extension.